esmvaltool/recipes/examples/recipe_check_obs.yml

```diff
@@ -578,6 +578,31 @@ diagnostics:
       - {dataset: ERA-Interim, project: OBS, tier: 3, type: reanaly, version: 1, start_year: 1979, end_year: 2018}
     scripts: null
 
+  ERA5:
+    description: ERA5 check
+    variables:
+      clt:
+      evspsbl:
+      evspsblpot:
+      mrro:
+      pr:
+      prsn:
+      psl:
+      rls:
+      rsds:
+      rsdt:
+      rss:
+      uas:
+      vas:
+      tas:
+      tasmax:
+      tasmin:
+      tdps:
+      ts:
+      tsn:
```

Comment:
See comment above, I understood that the cmorizer only supports 3 variables at the moment.